spotlight-client/src/assets/email-logo-white.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="14" height="14" viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Are these things we would want to include in the component library?

Reply:
yeah, probably? I should do a pass to move all of the icons in this application over to that package (if they're not there already) but I didn't want to block on that here.